.pre-commit-config.yaml

```diff
@@ -16,7 +16,6 @@ repos:
       - id: check-ast
       - id: check-json
       - id: check-merge-conflict
-      - id: check-yaml
```

Comment:
The linter fails on parsing the cloudformation template syntax, e.g. a `!Sub` function